tests/subsys/logging/log_cache/src/main.c

```diff
@@ -139,15 +139,15 @@ ZTEST(test_log_cache, test_log_cache_basic)
 
 	/* id0 is evicted since it is the oldest one. */
 	cache_get(&cache, id0.raw, &buf, false, __LINE__);
-	buf_check(buf, 1);
+	zassert_true(buf_check(buf, 2), "Buffer check failed");
```

Comment:
is it intentional to change  `buf_check(buf,1)` to `buf_check(buf, 2)`?

Reply:
yes, there is special commit for that